torch_geometric/testing/feature_store.py

```diff
@@ -37,6 +37,9 @@ def _get_tensor(self, attr: TensorAttr) -> Optional[FeatureTensorType]:
         if attr.index is None:
             return tensor
 
+        if isinstance(attr.index, torch.Tensor) and attr.index.numel() == 0:
```

Comment:
Can't we simply index with an empty index tensor here? Using `torch.tensor([])` is dangerous, in particular because the shape of the output should be `(0, num_features)` rather than `(0, )`.